src/pages/_app.tsx

```diff
@@ -10,23 +10,29 @@ import AppLayout from '@/shared/components/AppLayout';
 
 interface CustomPageProps {
   session: Session;
-  hideSidebars?: boolean;
+  hideLeftSidebar?: boolean;
```

Comment:
the graphs page only has a left sidebar, so we need to split up `hideSidebars` to have more modular control